src/app/services/data/courses.data.ts

```diff
@@ -64,33 +64,33 @@ export const courses: Course[] = [
   },
   {
     id: '4',
-    title: 'React JS course',
+    title: 'React',
     iconSrc: react,
     iconSmall: reactSmall,
     secondaryIcon: reactSecondary,
-    startDate: 'Jul, 2024',
-    language: ['ru', 'en'],
+    startDate: 'Jul 1, 2024',
```

Comment:
We're talking about date-libraries. I supposed we'll use it here - you have one data-source with date in `numeric` format and then we format it as we need depending on the component

Reply:
In issue https://github.com/rolling-scopes/site/issues/61, I plan to spell out the logic for working with dates and add a library to that block there. I can add this point to the issue.

Reply:
I suppose, libraries don't relevant to #61 task. You can add library in this PR to foment PR 61 task instead